app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -330,9 +282,39 @@ class Exposure extends React.Component {
   isolationFields = () => {
     return (
       <React.Fragment>
+        <Alert variant="warning">
+          You must enter a symptom onset date AND/OR a <u>positive</u> lab result to enroll this case
+        </Alert>
         <Form.Row>
-          <Form.Group as={Col} lg={{ span: 8, order: 1 }} md={{ span: 12, order: 1 }} xs={{ span: 24, order: 1 }} controlId="symptom_onset" className="mb-2">
-            <Form.Label className="nav-input-label">SYMPTOM ONSET DATE{schema?.fields?.symptom_onset?._exclusive?.required && ' *'}</Form.Label>
+          <Form.Group as={Col} xl={8} md={12} xs={24} controlId="symptom_onset" className="mb-2">
+            <Form.Label className="nav-input-label">
+              <React.Fragment>
+                SYMPTOM ONSET DATE
```

Comment:
Thoughts on reusing the `SymptomOnsetDate` component now that you've added all the stuff below? There are some differences between the enrollment input and the patient page input, but I'm thinking either the differences could be parameterized, and/or specifically [this section](https://github.com/SaraAlert/SaraAlert/blob/ea0697f83c29320b8930fb7a5dcf63015ea342fb/app/javascript/components/patient/assessment/actions/SymptomOnset.js#L123-L160) could be broken out into a separate reusable component.

Reply:
I think it might be simpler to keep it as is because the SymptomOnset component handles updating symptom_onset along with a good amount of other logic, which could potentially be subject to change in the future. We could parameterize the section you mentioned but I'm not sure if I'm a fan of having multiple symptom onset components because that could be confusing.

I think it made sense to refactor the FirstPositiveLaboratory component because there's a lot more reusable code there, but symptom onset is just a single field in the enrollment form

Reply:
Reasonable